src/lsp/cobol_parser/text_lexer.mli

```diff
@@ -40,6 +46,7 @@ val reserve_words: lexer -> Cobol_config.words_spec -> unit
 val enable_tokens: TokenHandles.t -> unit
 val disable_tokens: TokenHandles.t -> unit
 val decimal_point_is_comma: lexer -> lexer
+val specify_intrinsic: lexer -> string -> unit
```

Comment:
`val register_as_intrinsics: TokenHandles.t -> unit` (maybe `IntrinsicHandles.t` instead, if you want some more typing constraints over tokens). Missing `unregister_intrinsics: … -> unit` as well.
Or something along those lines. Possibly the only changes needed to this `mli`: `as_intrinsic` and `token_of_intrinsic_handle` should not be needed.